### DISCUSSION TITLE
When can we expect V1.0.0

### DISCUSSION THREAD

**Discussion by GregNowak** [ThrowTheSwitch/Ceedling#960] [General]
I was very excited to see Ceedling support Ruby 3, and even more excited in October with the announcement that an official release would be coming soon!   
I'm kicking off a new project and I'm currently using the 1.0.0 pre-release and having my team manually install the gem, but I'd love to be able to give an ETA on official support for 1.0.0.   

Do you have an expected date on the 1.0.0 release?

**Comment by jmrubillon** [comment]
It has kind of been answered [here](https://github.com/ThrowTheSwitch/Ceedling/discussions/825).
Soon...
It would make for a lovely Christmas present
